snykTask/src/task-args.ts

```diff
@@ -35,6 +35,8 @@ class TaskArgs {
   testDirectory: string | undefined = '';
   additionalArguments: string = '';
   ignoreUnknownCA: boolean = false;
+  // Snyk Code severity with its own pickList (no critical)
+  codeSeverityThreshold: string | undefined = '';
```

Comment:
suggestion: I'd rather have one argument with critical mapped to high if it's code.

Reply:
the introduction of `codeSeverityThreshold` is primarily for pipeline extension GUI editor to display 3 options at its combo box. it could be removed if it is fine to display the 4 options and handle mapping _critical_ option to _high_ in the extension.